lib/ruby_snowflake/client.rb

```diff
@@ -12,6 +12,12 @@
 require "securerandom"
 require "uri"
 
+begin
+  require "active_support"
```

Comment:
This only loads the extensions _required_ by ActiveSupport, so we need to add:

```suggestion
  require "active_support"
  require "active_support/notifications"
```

Reply:
fixed!